core/timers/timers.ts

```diff
@@ -31,12 +31,11 @@ export class Timers {
   static readonly memoryLocationDividerRegister: i32 = 0xff04; // DIV
   static dividerRegister: i32 = 0;
   static updateDividerRegister(value: i32): void {
-    let oldDividerRegister: i32 = Timers.dividerRegister;
-
+    // value is unused. Is it correct?
```

Comment:
So with all of the "old" values, really what I am trying to do is:

* Save the current value of the variable.
* Mutate the variable
* Compare the mutated variable, with its "old" state.

Thus in this case, I don't think it is unused? 🤔

Reply:
I mean you don't use input `value` param inside body. Is it mistake? Or `updateDividerRegister` don't need input argument?

Reply:
Ohhhhh! Haha! didn't realized I named the param value 😂 

Yeah it was probably unused. I copy and past a lot of things throughout my own code (as I am sure you can tell), and leads to code smells like this 🙃 .

Thanks for catching that!